web/static/js/components/simulation/VoiceWindow.jsx

```diff
@@ -74,25 +80,48 @@ const VoiceWindow = translate()(class extends Component<VoiceWindowProps> {
         this.playIVR(ivr)
       }
     } else {
-      this.spectrum.stop()
```

Comment:
We should still stop the spectrum, to avoid having a background timer being called at 60fps. `playIVR()` restarts it, and you can restart it just before calling `this.audio.play()` for the repeat.